sig-scalability/annual-report-2022.md

```diff
@@ -89,16 +107,16 @@ Include any other ways you measure group membership
 
 Operational tasks in [sig-governance.md]:
 
-- [ ] [README.md] reviewed for accuracy and updated if needed
-- [ ] [CONTRIBUTING.md] reviewed for accuracy and updated if needed
+- [x] [README.md] reviewed for accuracy and updated if needed
+- [x] [CONTRIBUTING.md] reviewed for accuracy and updated if needed
       (or created if missing and your contributor steps and experience are different or more
       in-depth than the documentation listed in the general [contributor guide] and [devel] folder.)
-- [ ] Subprojects list and linked OWNERS files in [sigs.yaml] reviewed for accuracy and updated if needed
-- [ ] SIG leaders (chairs, tech leads, and subproject owners) in [sigs.yaml] are accurate and active, and updated if needed
+- [x] Subprojects list and linked OWNERS files in [sigs.yaml] reviewed for accuracy and updated if needed
+- [x] SIG leaders (chairs, tech leads, and subproject owners) in [sigs.yaml] are accurate and active, and updated if needed
 - [ ] Meeting notes and recordings for 2022 are linked from [README.md] and updated/uploaded if needed
```

Comment:
Heh...
There should certainly be recordings from this year, but I guess they weren't uploaded...

@marseel - are there still somewhere around so that we could upload at least some of those?

Reply:
There weren't many meetings in 2022 as no one was actually coming- there were only 4 recordings in total.
This changed recently and since the beginning of March, many more people started coming to our sig meetings. Unfortunately, I am unable to add them as I don't have permission to yt playlist :disappointed:

Reply:
Sent dm with instructions on access to manage the plalylist 👍